salt/cache/redis_cache.py

```diff
@@ -203,6 +213,9 @@ def __virtual__():
 
 
 def init_kwargs(kwargs):
+    """
+    Effectively a noop. Return an empty dictionary.
+    """
     return {}
```

Comment:
Do we really need this function? Since it just returns an empty dictionary....

Just asking, since I didn't dig deep enough as to why.

Reply:
Unfortunately... yeah. It's part of the cache signature :facepalm: